src/ContentClient.ts

```diff
@@ -260,52 +256,15 @@ export class ContentClient implements ContentAPI {
       queryParams.set('fields', [fieldsValue])
     }
 
-    // Set legacy filters for get all deployments
```

Comment:
Is it safe to delete this now? Or should we wait?

Reply:
It's ok. All catalysts support `from` already, right?